.github/workflows/tests.yml

```diff
@@ -24,6 +24,24 @@ jobs:
         with:
           name: bazel-testlogs
           path: bazel-testlogs
+  build_virtual_frameworks:
+    # Build the entire tree with this feature enabled. Longer term, we'll likely
+    # consider merging this feature into the default behavior and can re-align
+    # the CI job
+    name: Build ( Virutal Frameworks )
```

Comment:
same test as the other PR so need to remove one of them

Reply:
The other PR has this one as a dependency, so once it is rebased it will be there 👍